README.md

```diff
@@ -21,3 +21,66 @@ or, if you are a `pip` users
 ```shell
 pip install gliderpy
 ```
+
+gliderpy aims to make querying and downloading glider data easier. Here is how one would build a query using erddapy:
+
+```shell
+from erddapy import ERDDAP
```

Comment:
I'm not 100% sure duplicating our docs in the README in this fashion is a good idea. The chances of them getting out of sync is high and this would be quite confusing for users. Maybe we can:
1. leave the README with a bar minimum example, or
2. autogenerate the README based on the doc notebook.

Reply:
ohh okay got it @ocefpaf  then what about the label thing ,i also thought of adding an colour bar as it was regarding temperature but not sure whether it's work or not so i haven't made any changes regarding it ! but i have written up some code based upon my idea can i paste the code here so you can review?

Reply:
And also i think we also need to mention pip install cartopy in the  readme docs @ocefpaf